extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -79,7 +79,19 @@ function render_block( $attributes ) {
 		return '<p>' . esc_html( $player_data->get_error_message() ) . '</p>';
 	}
 
-	return render_player( $player_data, $attributes );
+	/*
+	 * Get the block attributes checking if `$attributes`
+	 * is an array and it has defined a property.
+	 *
+	 * It handles the dual-behavior of argument
+	 * of the callback_render() function, which
+	 * was recently introduced by this Pull Request:
+	 * https://github.com/WordPress/gutenberg/pull/21467
+	 */
+	if ( ! empty( $attributes->attributes ) ) {
+		$attributes = $attributes->attributes;
+	}
+	return render_player( $player_data, $block_attributes );
```

Comment:
`$block_attributes` will be undefined now. This line can just be reverted

Reply:
I've kept your suggestion updating the argument in the function. My bad